docs/config.md

```diff
@@ -0,0 +1,21 @@
+### Configuration Setup and Runtime behavior for Apache Spark client
+Apache Spark comes with a lots of configuration properties which can be set. The list is pretty extensible as well.
+
+Passing each and every configuration in command line is not tenable, so Apache Spark supports the use of properties 
+files where the user can place most of the configuration for reuse across submissions. 
+
+In addition, the user can still add or override more configuration values in this file in the command line.
+
+For this Spark client, Canonical has come up with a rich set of options for the user to specify configuration properties and also override
+them in a spirit similar to Apache Spark.
```

Comment:
```suggestion
For this Spark client, Canonical has come up with a rich set of options for the user to specify configuration properties and also override
them in a spirit similar to Apache Spark.
```

Spark client provides a rich set of options for the user to specify configuration properties and also override
them in a spirit similar to Apache Spark.